Dijkstra Algorithm/Dijkstra.playground/Contents.swift

```diff
@@ -0,0 +1,137 @@
+//: Playground - noun: a place where people can play
+
+import Foundation
+import UIKit
+
+open class Vertex: Hashable, Equatable {
```

Comment:
Put this class in it's own file named `Vertex.swift`, and place it in the sources directory. The main page of the playground should contain only the demo code for clarity.